Executables/Game/PacketHandlers/Game/ItemUseHandler.cs

```diff
@@ -93,6 +100,48 @@ public async Task ExecuteAsync(GamePacketContext<ItemUse> ctx, CancellationToken
                     await player.SendItem(item);
                 }
             }
+        }else
+        {
+            switch (itemProto.Type)
+            {
+                case (byte) EItemType.Use:
+                    _logger.LogDebug("Use item");
+                    switch (itemProto.Subtype)
+                    {
+                        case (byte) EUseSubTypes.AbilityUp:
+                            _logger.LogDebug("Use ability up");
+                            var type = itemProto.Values[0];
+                            var duration = itemProto.Values[1];
+                            var value = itemProto.Values[2];
+                            var applyInfo = Enum.GetName(typeof(EApplyTypes), type);
+                            var applyType = (EPoints) Enum.Parse(typeof(EPoints), applyInfo);
+                            switch (type)
+                            {
+                                case (byte) EApplyTypes.MoveSpeed:
+                                    _ = _affectController.AddAffect(player, (int) EAffectTypes.AffectMoveSpeed, (int) applyType, value, (int) EAffectBits.MoveSpeedPotion, duration, 0);
+                                    break;
+                                case (byte) EApplyTypes.AttackSpeed:
+                                    _ = _affectController.AddAffect(player, (int) EAffectTypes.AffectAttackSpeed, (int) applyType, value, (int) EAffectBits.AttackSpeedPotion, duration, 0);
+                                    break;
+                            }
+                            break;
+                        case (byte) EUseSubTypes.UseSpecial:
+                            var specialItemType = EAffectTypes.None;
+                            var bonus = EPoints.None;
+                            var infDuration = 60 * 365 * 24 * 60 * 60;
+                            switch (itemProto.Id)
```

Comment:
Please add at TODO to remove the hardcoded IDs. It's okay for now but they must be handled elsewhere in the future